mph/discovery.py

```diff
@@ -332,14 +332,16 @@ def search_macOS():
         if not jre.exists():
             logger.error('Did not find Java run-time environment.')
             continue
-        java = jre/'bin'
+        java = jre/'Contents'/'Home'/'bin'
         if not java.exists():
             logger.error('Did not find Java run-time binaries.')
             continue
-        jvm = jre/'lib'/'amd64'/'server'/'libjvm.so'
+        jvm = jre/'Contents'/'Home'/'lib'/'server'/'libjvm.so'
         if not jvm.exists():
-            logger.error('Did not find Java virtual machine.')
-            continue
+            jvm = jre/'Contents'/'Home'/'lib'/'server'/'libjvm.dylib'
+            if not jvm.exists():
+                logger.error('Did not find Java virtual machine.')
+                continue

```

Comment:
Could the file extension really be either `.so` or `.dylib`, so that we have to check both?

Reply:
I actually don't know that and in this case it was a better safe than sorry. Ive seen *so and *dylib on my system, depending on the package

Reply:
Okay, then we'll leave it as is.